code-challenges.js

```diff
@@ -9,60 +9,155 @@
 
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
+        //To solve this, I begin with the accessor |.join| in order to combine the
+  // two arrays. This is outputing the arrays adding themselves together instead of my intended action.
+
+       //Realizing |.join| creates a string, I proceed with accessor |.concat|, successfully
+  //combining the arrays into one.
+
+      //Next, in order to return the length and not the parameters, I choose to create a variable
+  // called |combinePadres| to bring both constants together.
+
+      // Getting the output: |9| requires to console log my new variable, arguing a .length 
+  //built in method.
+
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+
+
+var combinePadres = padres1984WorldSeriesRuns. concat(padres1998WorldSeriesRuns)
+
+console.log(combinePadres.length)
+
 // Expected output: 9
 
+
+
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Describe your process:
+        //To solve this problem, I need to convert this string into an Array. I begin to set 
+    //up a console.log, next my goal is to use the array method |.split("")| to convert the 
+    //string "Golf 2022" into an array. Specifically using ("") as the argument to separate each
+    // individual letter.
+        //Once the string has been changed to an array, I can now freely reverse the letters in 
+    //the array with |.reverse|.
+        // My last step is to now bring the output back to the original string. I add a
+    //|.join| in order to accomplish this. With the code below, the outcome in terminal
+    //shows the intended answer.
+
+
 
 const currentCohort = "Golf 2022"
+
+console.log(currentCohort.split("").reverse().join()) 
+ 
 // Expected output: "2202 floG"
 
+
+
+
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Describe your process:
 
+          // To find last index of the provided values, I utilize the built in method array 
+    //accessor |.lastIndexOf|.
+          // Setting up console.log, I run the array |myNumbers| with the built in method |.lastOfIndex|.
+    // Const |givenValue1| will be the target. Running the code in terminal provides the expected
+    //output: 7.
+          //I use the exact same method to return the last index of const |givenValue2|, and 
+    //get the expected outut: 8.
+          //****** Even though I did get the correct answer using this straigh-forward method, 
+    //I don't quite understand why these ARE the answers. I'm counting the indexes and it is not 
+    //making sense the way |.lastIndexOf| is finding these answers.  
```

Comment:
LastIndexOf is checking a specific value against each value inside the array.  It will then tell you the index of the last instance that value appears in the array.